### PR TITLE
GET /v1/contextEntities/type/{entity::type}/id/{entity::id} 

### PR DESCRIPTION
**Pull request by kzangeli**
One more convop converted to using std-op

### CODE REVIEW

File: src/lib/common/statistics.cpp [hunk 1]
```diff
@@ -192,7 +193,7 @@ void statisticsUpdate(RequestType request, Format inFormat)
   case RtUnsubscribeContextResponse:                     ++noOfRtUnsubscribeContextResponse; break;
   case RtSubscribeResponse:                              ++noOfRtSubscribeResponse; break;
   case RtSubscribeError:                                 ++noOfRtSubscribeError; break;
-
+  case RtContextElementResponse:                         ++noOfContextElementResponse; break;
```
Comment (fgalan) [inline src/lib/common/statistics.cpp]: So one counter was missing until now and this is the bugfix? Need to mention in CHANGES_NEXT_RELEASE or in this case is a very tiny bugfix?

File: src/lib/serviceRoutines/getAllEntitiesWithTypeAndId.cpp [hunk 2]
```diff
@@ -39,6 +41,18 @@
...
 * getAllEntitiesWithTypeAndId - 
+*
+* GET /v1/contextEntities/type/{entity::type}/id/{entity::id}
+* GET /ngsi10/contextEntities/type/{entity::type}/id/{entity::id}
+*
+* Payload In:  None
+* Payload Out: ContextElementResponse
+* 
+* URI parameters:
+*   - attributesFormat=object
```
Comment (fgalan) [inline src/lib/serviceRoutines/getAllEntitiesWithTypeAndId.cpp]: In this case limit/offset/details don't make sense, do they? Almost sure at 99% percent, just to be sure...

Comment (kzangeli) [inline src/lib/serviceRoutines/getAllEntitiesWithTypeAndId.cpp]: Yeah, it's only one output from this request (ContextElementResponse - not a vector).
There is nothing to count.

File: src/lib/common/statistics.cpp (hunk 1 above)
Comment (kzangeli) [inline src/lib/common/statistics.cpp]: I only needed this constant for a call to ContextElementResponse::render.
As a side-effect it is added as a counter in statistics.
However, as I haven't included it in the response to a statistics request, an external user will notice no difference.
No big deal really. What we should do is to look into the statistics some day soon ...

Comment (fgalan) [inline src/lib/common/statistics.cpp]: Ok, NTC.

File: src/lib/serviceRoutines/getAllEntitiesWithTypeAndId.cpp (hunk 2 above)
Comment (fgalan) [inline src/lib/serviceRoutines/getAllEntitiesWithTypeAndId.cpp]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**